include/mbedtls/x509_csr.h

```diff
@@ -205,6 +205,14 @@ void mbedtls_x509write_csr_set_md_alg( mbedtls_x509write_csr *ctx, mbedtls_md_ty
  * \param key_usage key usage flags to set
  *
  * \return          0 if successful, or MBEDTLS_ERR_X509_ALLOC_FAILED
+ *
+ * \note            The <code>decipherOnly</code> flag from the Key Usage
```

Comment:
Minor: Can we use `\c` here to ease reading?

Reply:
I understand what you mean, but I would rather not do that because then we have to go through all the review process again just to change this... I think these code tags are acceptable and they are actually used in other parts of the library too.

Reply:
Yes, fine by me.